mongodbatlas/fw_resource_mongodbatlas_alert_configuration.go

```diff
@@ -709,9 +713,6 @@ func newTFNotificationModelList(matlasSlice []matlas.Notification, currStateNoti
 		if !currState.EmailAddress.IsNull() {
 			newState.EmailAddress = conversion.StringNullIfEmpty(value.EmailAddress)
 		}
-		if !currState.IntervalMin.IsNull() {
-			newState.IntervalMin = types.Int64Value(int64(value.IntervalMin))
-		}
```

Comment:
I was reading this [plan-modification#plan-modification-process](https://developer.hashicorp.com/terraform/plugin/framework/resources/plan-modification#plan-modification-process)
> Plan Modification Process
> When the provider receives a request to generate the plan for a resource change via the framework, the following occurs:
> 
> Set any attributes with a null configuration value to the [default value](https://developer.hashicorp.com/terraform/plugin/framework/resources/default).
> If the plan differs from the current resource state, the framework marks computed attributes that are null in the configuration as unknown in the plan. This is intended to prevent unexpected Terraform errors. Providers can later enter any values that may be known.
> Run attribute plan modifiers.
> Run resource plan modifiers.

I am wondering if `currState.IntervalMin.isNill` is `false`  but `currState.IntervalMin.IsUnknown()` is `true` when calling `terraform plan`. Could you verify? if that is the case we should add `currState.IntervalMin.IsUnknown()` in all of this ifs

Reply:
Yes your statement is correct, I could verify that the attribute in `currState` comes as unknown as it is an undefined computed attribute.

In this case, since it is a value that is always defined in the api request/response (defined as an int in manual sdk), we can simply define the value in the model. This is what solves the inconsistency issue when doing the import operation, because when importing you have no state/config to verify if field was defined by the user, so value coming from the api is set directly.